roles/openshift_common/tasks/main.yml

```diff
@@ -4,6 +4,9 @@
   hostname: name={{ openshift_bind_ip }}
   when: openshift_hostname_workaround
 
+- name: Update all packages
+  yum: name=* state=latest
+
```

Comment:
@jwhonce that sounds like a good plan. Please name the role generically so that we can use it in other places.

I'm thinking something like "os_update"